src/data/ItemTemplate.json

```diff
@@ -1957694,12 +1957775,12 @@
     "Bonus1": 3,
     "Bonus10": 0,
     "Bonus10Type": 0,
-    "Bonus1Type": 3,
-    "Bonus2": 1,
-    "Bonus2Type": 1,
-    "Bonus3": 1,
-    "Bonus3Type": 18,
-    "Bonus4": 0,
+    "Bonus1Type": 0,
```

Comment:
I see a bunch of these bonus types being set to 0, you sure about this?

Reply:
Yeah, that's so right.